libs/ui/src/lib/ui.tsx

```diff
@@ -129,7 +129,12 @@ export const Ui = (props: UiProps) => {
       }
     }
 
-    const formDataClone = structuredClone(data.formData)
+    let formDataClone = structuredClone(data.formData)
+    const priorTreatmentRequirementNames = formDataClone.prior_treatment_requirements?.prior_treatment_requirement && formDataClone.prior_treatment_requirements?.prior_treatment_requirement.map((item: { prior_treatment_requirement_name: any; }) => item.prior_treatment_requirement_name) || formDataClone.prior_treatment_requirements?.prior_treatment_requirement;
```

Comment:
I don't understand the point of this check, if "formDataClone.prior_treatment_requirements?.prior_treatment_requirement" is defined, then map of that (even if empty array) will always return a defined array

Reply:
Here before mapping I am checking whether formDataClone.prior_treatment_requirements?.prior_treatment_requirement is there in the data or not because when the value is not defined and if we try to map it we are getting an undefined error